src/starbase/setup.ts

```diff
@@ -2,10 +2,17 @@ import { Clone, Reference, Repository } from 'nodegit';
 import { StarbaseConfig, StarbaseIntegration } from './types';
 import { isDirectoryPresent } from '@jupiterone/integration-sdk-runtime';
 import { executeWithLogging } from './process';
+import camelCase from 'lodash.camelcase';
+import mapkeys from 'lodash.mapkeys';
+import Ajv,  { Schema } from 'ajv';
+
+const ajv = new Ajv();
+ajv.addKeyword("mask");
 
 async function setupStarbase(config: StarbaseConfig) {
   for (const integration of config.integrations) {
     await setupIntegration(integration);
+    await checkInstanceConfigFields(integration);
```

Comment:
Thoughts on exposing a sep command to validate your config without running `setup`? Doesnt need to go in this PR, just throwing the idea out.